apps/frontend/src/AuthenticatedApp.tsx

```diff
@@ -64,20 +64,20 @@ const AuthenticatedApp: FC<Record<string, never>> = () => {
   return (
     <Onboardable>
       {(onboardable) => (
-        <Layout
-          userOnboarded={user.onboarded}
-          onboardable={onboardable}
-          onboardModalHref={
-            tabRoute ? tabRoute({}).editOnboarded({}).$ : undefined
-          }
-          userProfileHref={network({}).users({}).user({ userId: user.id }).$}
-          teams={user.teams.map(({ id, displayName = '' }) => ({
-            name: displayName,
-            href: network({}).teams({}).team({ teamId: id }).$,
-          }))}
-          aboutHref="https://www.parkinsonsroadmap.org/"
-        >
-          <CheckOnboarded>
+        <CheckOnboarded>
```

Comment:
whats the difference in changing the order of the components?

Reply:
@jhonny-m  on the first version, I added a BrowserRoute on the CheckOnboarded component that was blocking the events on the OnboardingFooter. On the last version, we don't need any of that